.github/workflows/push.yml

```diff
@@ -579,6 +579,51 @@ jobs:
       with:
         name: components-vue-vrt-regression
         path: ./packages/components-vue/tests/vrt/results
+  crawler-e2e-puppeteer:
```

Comment:
This job should probably become a condition for other jobs to start? You can search for `needs:` within this file to find some.

Reply:
Added it as a condition for Storefront and Assets deployments